### ISSUE TITLE
ADMIN: New parameter "pet type" is not listed as an option under flexible form designer (Admin Bug 006)

### ISSUE THREAD

**Issue by NatalieShapiro** [EIDSS/EIDSS7#294]
**Summary**
Verification steps of ADM10 Administration cannot be completed. "Pet type" is not showing up as an option for a parameter in the flexible form design. 

**Details**
Script ID (or enter ad hoc if ad hoc): ADM10 Administration
Username: NCDCAdmin
Severity (optional):

**To Reproduce**
Steps to reproduce the behavior (if in scripted testing, can refer to script step):

1. Go to Flexible form designer, parameters tab, try to add new parameter - look for "pet type" under "type"

**Expected behavior**
Expect to see pet type as an option.

**Screenshots**

**Additional context**

**Comment by jkkannan** [comment]
Known issue; check devops

**Comment by karlasomera** [comment]
Able to run through Script.  I was able to add  a new fixed Preset Value as Pet Type #25.  added the three values then under the flexible form designer, I added the new parameter to add the new fixed preset Value (PetType #25).  This bug I recommend to close since he behavior of this script was to create a new Parameter type with customized fixed Values.  The only concern that I have was not able to see the values that were entered in the parameter types (Hedgehog, Turtle, and Snake).
